unstract/connectors/src/unstract/connectors/filesystems/google_drive/google_drive.py

```diff
@@ -95,3 +96,12 @@ def test_credentials(self) -> bool:
         except Exception as e:
             raise ConnectorError(str(e))
         return True
+
+    @staticmethod
+    def get_connector_root_dir(input_dir: str, **kwargs: Any) -> str:
+        """Get roor dir of gdrive."""
+        root_path = kwargs.get("root_path")
+        if root_path is None:
```

Comment:
Is the `root_path` being fetched from the user? If not, can we not show this error message to the user? Instead we can give some user friendly message, helping the user to take actions.

Reply:
@harini-venkataraman  we have `root_path = root` enabled for all connectors internally. I am just using root_path for GDrive, other's don't even this. Its not being fetched from user.